src/timeseriesinsights/azext_timeseriesinsights/commands.py

```diff
@@ -0,0 +1,104 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-locals
+from azure.cli.core.commands import CliCommandType
+from azure.cli.core.commands.transform import gen_dict_to_list_transform
+
+
+def load_command_table(self, _):
+
+    with self.command_group('timeseriesinsights', is_experimental=True) as g:
+        pass
+
+    from ._client_factory import cf_operations
+    timeseriesinsights_operations = CliCommandType(
+        operations_tmpl='azext_timeseriesinsights.vendored_sdks.timeseriesinsights.operations._operations_operations#OperationsOperations.{}',
+        client_factory=cf_operations)
+    with self.command_group('timeseriesinsights operation', timeseriesinsights_operations, client_factory=cf_operations) as g:
+        g.custom_command('list', 'list_timeseriesinsights_operation')
+
+    # region environment
+    from ._client_factory import cf_environments
+    timeseriesinsights_environments = CliCommandType(
+        operations_tmpl='azext_timeseriesinsights.vendored_sdks.timeseriesinsights.operations._environments_operations#EnvironmentsOperations.{}',
+        client_factory=cf_environments)
+
+    with self.command_group('timeseriesinsights environment standard', timeseriesinsights_environments, client_factory=cf_environments) as g:
+        g.custom_command('create', 'create_timeseriesinsights_environment_standard',
```

Comment:
Need support no wait as it's a LRO.

Reply:
Good point, but `update` is currently broken on the service side: https://github.com/Azure/azure-rest-api-specs/issues/9265. Let's `supports_no_wait` anyway.